spec/features/register_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+feature 'user registration', {vcr: true, js: true} do
+
+  background do
+    visit register_path
+  end
```

Comment:
For small blocks, `{}` is used rather than `do` / `end` which is used for multilines block.